Add template for requesting a new command (#9)
svthalia/discord-bot: .github/ISSUE_TEMPLATE/request-command.md

---

> name: Command request
> about: A new command should be added
> labels: 'command, priority: low'

Reviewer: ```suggestion
name: Command request
title: "Add new command: [NAME]"
about: A new command should be added
labels: 'feature, command, priority: low'
```

---

> ### Command [new]
> #### Name

Reviewer: ```suggestion
### Name
```

> how the command is invoked `[prefix] [command] [args]`

Reviewer: ```suggestion
How the command is invoked `[prefix] [command] [args]`?
```

> ####Description

Reviewer: ```suggestion
### Description
```

> what does the command do, what should `[prefix] help [command]` return to the user

Reviewer: ```suggestion
What does the command do and what should `[prefix] help [command]` return to the user?
```

> ####Use cases

Reviewer: ```suggestion
### Use cases
```

> How do you expect the command will be used

Reviewer: ```suggestion
How do you expect the command will be used?
```

> ####Permissions

Reviewer: ```suggestion
### Permissions
```

> Who has acces to the command

Reviewer: ```suggestion
Who should have access to the command?
```
